lib/metanorma/compile_validate.rb

```diff
@@ -29,31 +29,33 @@ def load_flavor(stdtype)
         Util.log("[metanorma] Info: Loading `#{flavor}` gem "\
                  "for standard type `#{stdtype}`.", :info)
       end
-      require_flavor(flavor, stdtype)
+      require_flavor(flavor)
       unless @registry.supported_backends.include? stdtype
         Util.log("[metanorma] Error: The `#{flavor}` gem "\
                  "still doesn't support `#{stdtype}`. Exiting.", :fatal)
       end
     end
 
-    def require_flavor(flavor, stdtype)
+    def require_flavor(flavor)
       require flavor
       Util.log("[metanorma] Info: gem `#{flavor}` loaded.", :info)
-    rescue Gem::ConflictError
-      Util.log("[metanorma] Error: Couldn't resolve dependencies for "\
-               "`metanorma-#{stdtype}`, Please add it to your Gemfile "\
-               "and run bundle install first", :fatal)
-    rescue LoadError
+    rescue LoadError => e
       msg = <<~MSG
-        [metanorma] Error: loading gem `#{flavor}` failed. Exiting.
+        Error: #{e.message}
+        An exception with Metanorma has occurred.
 
-        Troubleshooting:
-        1. If you are using metanorma via bundler/ruby, make sure that your
-           Gemfile contains a line:
-             gem "metanorma-#{stdtype}"
+        If this problem persists, please report this issue at this link and we shall respond
```

Comment:
"If this problem persists, please report this issue at this link" => "If this problem persists, please report this issue at the following link"

Reply:
Fixed